### DISCUSSION TITLE
今天教大家如何进入绳网

### DISCUSSION THREAD

**Discussion by ZZZ-MHY** [share121/inter-knot#1898] [其他]
![微信图片_20240813174130](https://github.com/user-attachments/assets/4f75131f-c8dc-4a50-8489-88551db83cc9)

教程:1.先下载Steam++，浏览器接搜就行，选Github加速。
2.打开视频里github.com/share121/inter-knot，然后注册登录Github用户，登录完会跳到自己的页面，再输一次github.com/share121/inter-knot就行，网页右边有一个绳网，下面是链接，打开然后它会提醒你安装脚本，点进去以后不要动。
3.浏览器搜索tampermonkey，然后找到添加这个浏览器插件就行，然后回到绳网助手页面，点击安装，后面油猴就会跳出来直接安装就行。
4.回到绳网，应该就进去了。

**Comment by CloudSwordSage** [comment]
谢谢，我学废了\\dog
![image](https://github.com/user-attachments/assets/181cf8a7-90af-42fc-841b-5e57a63a7aed)

**Comment by HDTianRu** [comment]
不懂，我用clash的

**Reply by LKBaka** [reply]
+1

**Reply by NekoWise** [reply]
我用的Zero Trust...

**Reply by HifumiMizuhara** [reply]
哈哈我也Clash

**Reply by gwmmdjh** [reply]
+1

**Reply by Regionnor** [reply]
我用的是v2rayN

**Reply by Shuyluo** [reply]
已普遍理性而言，你可以再水一帖(不是

**Reply by MuXue1230** [reply]
> 1
> > 2
> > > 3

**Comment by gaojiaobei** [comment]
谢谢楼主！在绳网看到了进入绳网的教学，让我学会了如何进入绳网，真是受益颇丰啊

**Comment by Enigma-Soul** [comment]
dev-sidecar 活了，没死，最为一个3年老用户，他很好用，https://github.com/docmirror/dev-sidecar

**Comment by lcshxl** [comment]
很有用的帖子，以前我根本不知道怎么进入绳网，现在终于知道了👍

**Comment by NekoWise** [comment]
不懂欸，所以到底怎么金绳网捏<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/40f465ef5312e4ba0b19c596b9999cef_8760502457253372865.png" alt="三月七 无语" width="75" >

**Comment by HifumiMizuhara** [comment]
问题是，不进绳网怎么看这条评论（捂脸）

**Reply by xnpboke** [reply]
并不是这样的，这条评论来自GitHub，而不是在绳网上发的（

**Comment by NaNo-yuo** [comment]
在绳网写进入绳网的攻略，真是太实用啦<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by WHKKa** [comment]
谢谢，已经上来了<img src="https://img-static.mihoyo.com/communityweb/upload/54f6c5f6a4bf851877158b0000e636c5.png" alt="克拉拉 给你心心" width="75" >

**Comment by Genshin-Neuvillette** [comment]
有没有可能能看到你这篇帖子的都是已经进来的

**Comment by Ashphi123** [comment]
感谢楼主！原来我之前上的都是假绳网！

**Comment by anything12121** [comment]
震惊！！！原来我已经进入绳网了！？我一直都不知道啊？？？

**Comment by MuTangIII** [comment]
也是进入绳网了，感谢老师~

**Comment by shuzhide** [comment]
太有用了，终于进绳网了/dog

**Comment by soyobat** [comment]
去b站问如何下载B站

**Comment by Hitokui12138** [comment]
太棒了,就像在火车上教别人怎么买火车票一样有用!

**Comment by ysd1911** [comment]
原来我进来了吗？
但是这个页面好像不太对啊
QAQ

**Comment by MrDucker** [comment]
谢谢我已经进去了<img src="https://bbs-static.miyoushe.com/static/2023/05/29/87613981dfd3b3ec11622280857f9cad_342243593322755411.png" alt="兰那罗-飞行" width="75" >

**Comment by kiana127127127** [comment]
今天终于知道怎么上真绳网了

**Comment by qwq775** [comment]
谢谢楼主！在绳网看到了进入绳网的教学，让我学会了如何进入绳网，真是受益颇丰啊

**Comment by Tuskiy** [comment]
谢谢楼主，终于知道怎么上真绳网了

**Comment by JobNumber-9527** [comment]
看不懂啊，看来我上不了绳网了[大哭]

**Comment by fazhize** [comment]
学费了，学费了

**Comment by yuxigongzi** [comment]
不知道，用“interknot.work”进来的，话说回复怎么还是在gayhub界面回复

**Reply by JobNumber-9527** [reply]
不是哥们“gay”hub是什么玩意

**Reply by Enigma-Soul** [reply]
他说的没问题   Github有个外号叫全球最大的同性恋交流网站（因为程序员大部分都是男的）  所以翻译过来很正常，叫Gayhub

**Reply by Enigma-Soul** [reply]
你8月16日加入Github不知道也能说得通

**Comment by jiangwuchang** [comment]
有没有一种可能，已经进来了

**Comment by fa-e-tong-LKN** [comment]
按照楼主的步骤做了一遍，怎么又到这里了

**Reply by ysd1911** [reply]
这里就是绳网

**Reply by fa-e-tong-LKN** [reply]
坏了，我下次一定记得手动狗头

**Comment by Maple013** [comment]
-我要怎么进入绳网呢？
*去看ZZZ-MHY的教程啊。
-那我要去哪里看ZZZ-MHY的教程呢？
*到绳网上去看啊。
-……
-诚邀搬运工，将绳网发布的教程私信给我，价格免议。

**Comment by yigexie** [comment]
是真的！我成功了！
<img src="https://img-static.mihoyo.com/communityweb/upload/4a4ca879edac9bf65768a796305abdbd.png" alt="哼哼" width="75" >

**Comment by 16632910773** [comment]
总感觉哪里不对，啊啊啊要长脑子了

**Comment by yivea** [comment]
谢谢，太有用了

**Comment by git1145146899** [comment]
<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by chunqiu-ui** [comment]
谢谢楼主，受益匪浅，终于是用上绳网了

**Comment by jlnof** [comment]
谢谢楼主，已经用上了

**Comment by dalizi2333333** [comment]
谢谢楼主，以前用的都是假绳网，现在终于用上真·绳网了

**Comment by imacatperson** [comment]
卡在第二步，链接打不开了怎么办<img src="https://img-static.mihoyo.com/communityweb/upload/ee7bb800b0c90f66fa9e321cb6d8ee17.png" alt="抱大腿" width="75" >

**Comment by wazds** [comment]
笑死，我也是进去了

**Comment by yinyue0zzz** [comment]
![image](https://github.com/user-attachments/assets/916f2e80-9a75-43d4-8cc1-b792cf14ac39)
链接打不开怎么办
